Extensions/PrimitiveDrawing/Extension.cpp

```diff
@@ -116,47 +135,47 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
   obj.AddAction("Star",
                 _("Star"),
                 _("Draw a star on screen"),
-                _("Draw at _PARAM1_;_PARAM2_ a star with _PARAM3_ points and radius: _PARAM4_ (inner radius: _PARAM5_, rotation: _PARAM6_ ) "
+                _("Draw at _PARAM1_;_PARAM2_ a star with _PARAM3_ points and radius: _PARAM4_ (inner radius: _PARAM5_, rotation: _PARAM6_) "
                 "with _PARAM0_"),
                 _("Drawing"),
                 "res/actions/star24.png",
                 "res/actions/star.png")
 
       .AddParameter("object", _("Shape Painter object"), "Drawer")
-      .AddParameter("expression", _("X coordinate of the center of the star"))
-      .AddParameter("expression", _("Y coordinate of the center of the star"))
+      .AddParameter("expression", _("X position of center"))
+      .AddParameter("expression", _("X position of center"))
       .AddParameter("expression", _("Number of points of the star (minimum: 2)"))
       .AddParameter("expression", _("Radius (in pixels)"))
       .AddParameter("expression", _("Inner radius (in pixels, half radius by default)"))
       .AddParameter("expression", _("Rotation (in degrees)"))
       .SetFunctionName("DrawStar")
       .SetIncludeFile("PrimitiveDrawing/ShapePainterObject.h");
 
-  // These actions are not exposed yet as the way they work is unsure. See https://github.com/4ian/GDevelop/pull/1256
-/*
+
   obj.AddAction("Arc",
                 _("Arc"),
                 _("Draw an arc on screen"),
-                _("Draw at _PARAM1_;_PARAM2_ an arc with _PARAM3_ of radius, start angle: _PARAM4_, end angle: _PARAM5_ (anticlockwise: _PARAM6_) "
+                _("Draw at _PARAM1_;_PARAM2_ an arc with _PARAM3_ of radius, start angle: _PARAM4_, end angle: _PARAM5_ (anticlockwise: _PARAM6_, close path: _PARAM7_) "
```

Comment:
I think "x of radius" is not proper english, let's use instead "[...] an arc with radius: _PARAM3_, start angle: [...]"